index.js

```diff
@@ -2,7 +2,7 @@
 const parseDataUrl = require('parse-data-url');
 
 function constructRequest(har, userAgent = false) {
-  if (!har) throw new Error('Missing HAR file');
+  if (!har) throw new Error('Missing HAR definition');
```

Comment:
Does this make it a major version bump?

Reply:
Yeah this'll be a major bump because of the header fix and also some other dev deps I updated.

Reply:
And I'm also dropping support for Node 10*